04-assorted-topics-in-java.md

```diff
@@ -183,7 +183,7 @@ Here, we did the following changes:
 2. Everywhere we used `Object`, we switched it with `T`.
 3. When we create an instance of `MyItem`, we specify that `T` should be `Integer`.
 
-Note the mapping in syntax between `MyItem<T>` and `MyItem<Integer>`! Everywhere we use `T` in our class, we can pretend it says `Integer` for this instance. Furthermore, we don't need to do any casting as we know `mi.item` will be of typeInteger since `mi` is of type `MyItem<Integer>`. We can make new `MyItem` objects with other types too, if we'd like!
+Note the mapping in syntax between `MyItem<T>` and `MyItem<Integer>`! Everywhere we use `T` in our class, we can pretend it says `Integer` for this instance. Furthermore, we don't need to do any casting as we know `mi.item` will be of type Integer since `mi` is of type `MyItem<Integer>`. We can make new `MyItem` objects with other types too, if we'd like!
```

Comment:
Oh, could you add the ` ` around Integer too? Then We can merge it in. Thanks for catching this.

Reply:
My pleasure! I've added the ` quotes as requested.